pennylane/transforms/__init__.py

```diff
@@ -12,10 +12,50 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """
-This subpackage contains tape and QNode transforms.
+This subpackage contains QNode, quantum function, device, and tape transforms.

```

Comment:
We can expand this section down the line with more details and examples on how to write your own transform